assets/scss/main-menu.scss

```diff
@@ -589,3 +589,7 @@
     }
   }
 }
+// On the first level, to let override this rule in a custom theme or module.
+.main-menu-link--level-3 {
+  display: none;
```

Comment:
Should we do the opposite in small_y?
I mean to display main-menu-link--level-3 according to the design.

Reply:
It's already there :)